chocoban/pc-backend
# updates master with develop

What does this PR do?
- Set up project
Description of Task to be completed?
- Set up project
How should this be manually tested?
- Checkout to ch-setup-project branch
- Check .env.example for environmental variables
- On terminal, run command yarn start:dev
Any background context you want to provide?
N/A

What are the relevant pivotal tracker stories?
Postman Documentation
N/A

Screenshots (If applicable)
N/A
